css/style.css

```diff
@@ -0,0 +1,116 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-image: url("../Image/Background.svg");
+  background-repeat: no-repeat;
+  background-size: cover;
```

Comment:
- [ ] [Optional] Please add this background image to the headline section, not to the whole body, and position it well to match the given design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/170707949-43c75cb8-6593-4da6-a4e3-798aeb798882.png)  | ![image](https://user-images.githubusercontent.com/68796232/170707244-05efc6cc-f791-4dd5-9082-51306fafaf08.png)|